src/context/index.ts

```diff
@@ -19,23 +20,67 @@ const nonPrimitiveProps = [
   'INCLUDED_PROPS'
 ];
 
-export default async function(config) {
+type Config = {
```

Comment:
First pass at typing the config object, just enough to get compilation to pass. There is another Jira ticket to flesh out configuration a bit more later.